src/components/checkbox/checkbox.template.tsx

```diff
@@ -10,7 +10,9 @@ interface CheckboxTemplateProps {
     label?: string;
 }
 
-export const CheckboxTemplate: FunctionalComponent<CheckboxTemplateProps> = props => {
+export const CheckboxTemplate: FunctionalComponent<CheckboxTemplateProps> = (
+    props
+) => {
```

Comment:
This is a little unfortunate. I assume it's an indirect result of the added parenthesis, as the added line length probably caused Prettier to break the line. Not the end of the world though, imo.